File: sample/src/main/java/com/davidmiguel/sample/Card2Activity.java
package com.davidmiguel.sample;

import android.os.Bundle;
import android.view.View;

import com.davidmiguel.dragtoclose.DragToClose;

import androidx.appcompat.app.AppCompatActivity;

/**
 * In this example the card can be dragged out card but the activity is not closed.
 * A label allows to reopen the card.
 */
public class Card2Activity extends AppCompatActivity {

    DragToClose dragToClose;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_card2);
        dragToClose = findViewById(R.id.drag_to_close);
    }

    public void openCard(View view) {
        dragToClose.openDraggableContainer();
    }
}

File: sample/src/main/java/com/davidmiguel/sample/Card2Activity.kt
package com.davidmiguel.sample

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.davidmiguel.dragtoclose.DragToClose

/**
 * In this example the card can be dragged out card but the activity is not closed.
 * A label allows to reopen the card.
 */
class Card2Activity : AppCompatActivity() {

    private lateinit var dragToClose: DragToClose

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_card2)
        dragToClose = findViewById(R.id.drag_to_close)
    }

    @Suppress("UNUSED_PARAMETER")
    fun openCard(view: View) {
        dragToClose.openDraggableContainer()
    }
}
